### PR TITLE
feat(umaverse): change grid structure

### PR DESCRIPTION
**Pull request by Gamaranto**
**Motivation**
This PR collapses the grid into 2 columns instead of 4

Before:
CleanShot 2021-08-24 at 11 40 41@2x

After:
CleanShot 2021-08-24 at 11 39 39@2x

https://app.clubhouse.io/uma-project/story/1929/rework-the-grid-for-the-lsp-page

**Details**
To make this change without introducing a lot of complexity or changing the markup dynamically, I've changed the order of components of the grid on mobile. 

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [x]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

File: components/Information.tsx [hunk 1]
```diff
@@ -9,16 +9,17 @@ import DuplicateIcon from "../public/icons/duplicate.svg";
...
+  | { className?: string; synth: Synth<{ type: "emp" }> };
+export const Information: React.FC<Props> = ({ synth, className }) => {
   const handleCopyClick = (text: string) => {
     window.navigator.clipboard.writeText(text);
   };
 
   return (
-    <Wrapper>
+    <Wrapper className={className}>
       <Heading>Token Information</Heading>
```
Comment (Tulun) [inline components/Information.tsx:23]: is className optional here?

**Review by Tulun — APPROVED** [review]
Minor nit.

It's ok, but I think this will need a large refactor sooner than later.

**Review by daywiss — APPROVED** [review]
im having a hard time understanding how you achieved this with the lsp view, while the code shows that you mainly changed the emp view.  otherwise though it looks like you got the right result

File: components/Information.tsx (hunk 1 above)
Comment (Gamaranto) [reply components/Information.tsx:23]: yes

**Pull request merged**